google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -40,6 +42,7 @@ private SessionPoolOptions(Builder builder) {
     // maxSessions value is less than the default for minSessions.
     this.minSessions = Math.min(builder.minSessions, builder.maxSessions);
     this.maxSessions = builder.maxSessions;
+    this.incStep = builder.incStep;
```

Comment:
Do we need to allow users to configure this? I mean we add a new parameter to let users to configure, which can add cognitive load to them.

Reply:
No, the configuration parameter is package-private, meaning that it can only be configured internally by the client library and its tests. Users can't see this configuration, unless they browse the source code of the client library.

It needs to be configurable for the benchmarks to be able to test different scenarios.